contracts/main/interfaces/IPriceFeed.sol

```diff
@@ -7,4 +7,6 @@ interface IPriceFeed {
     function peekPrice() external returns (bytes32, bool); // [wad]
 
     function isPriceOk() external view returns (bool);
+
+    function poolId() external view returns (bytes32);
```

Comment:
now each price feed gonna have poolId to be sure that we use it properly and make price feed dedicated to some particular pool

Reply:
this is such a good way to prevent confusion.